### ISSUE TITLE
L’ASL di Alessandria lancia un appello attraverso le pagine dei giornali e dei social media: servono

### ISSUE THREAD

**Issue by ehack-italy** [emergenzeHack/covid19italia_segnalazioni#2287]
<pre><yamldata>
Cosa: Asl di Alessandria cerca con urgenza medici anestesisti
Descrizione: 'L’ASL di Alessandria lancia un appello attraverso le pagine dei giornali
  e dei social media: servono medici anestesisti, anche in pensione. Le Rianimazioni
  sono in emergenza, implementiamo il numero di posti letto perché aumentano i pazienti
  gravi che devono essere intubati, ma abbiamo bisogno di personale medico specializzato
  in Anestesia e Rianimazione.

  Aiutateci a diffondere questo importante messaggio per salvare delle vite.

  L’ufficio assunzioni dell’ASL AL è contattabile telefonicamente al 0143 332290.'
Destinatari: Medici anestesisti
Indirizzo: 45.020793 8.615985 0 0
Link: http://www.aslal.it/appello-anestesisti
Nome: Asl Alessandria
Telefono: 0143 332290
</yamldata></pre>

**Comment by luciaroma** [comment]
@cristigalas @chiccap lanciamo sui social? Non ricordo .... abbiamo per caso budget per targettizzare i post?
e ... dove va regione_manuale? :-)

**Comment by cristigalas** [comment]
@luciaroma attenzione, la dicitura **_regione_manuale: tutte_**  era messa male. Va inserita in fondo, prima di _</yamldata></pre>_ ma in una nuova riga, come facciamo per la posizione. 
Esempio:
_Telefono: 0143 332290
Posizione: 44.83495335 8.745030418605866 0 0
**regione_manuale: tutte**
</yamldata></pre>_

Ma dobbiamo decidere cosa fare per le richieste di aiuto, soprattutto di questo tipo... @chiccap tu cosa ne pensi?

**Comment by luciaroma** [comment]
in realtà l'avevo messa lì dove dici tu, ma il bot mi ha tolto accettato e quindi ho pensato andasse messa altrove. Fortini poi mi ha detto di inserirla qui dentro </yamldata> e in effetti poi è andata.

**Comment by luciaroma** [comment]
"Ma dobbiamo decidere cosa fare per le richieste di aiuto, soprattutto di questo tipo... @chiccap tu cosa ne pensi?"

Cosa intendi? Come gestirle in accettazione o come tentare di dare risposta?

**Comment by cristigalas** [comment]
> "Ma dobbiamo decidere cosa fare per le richieste di aiuto, soprattutto di questo tipo... @chiccap tu cosa ne pensi?"
> 
> Cosa intendi? Come gestirle in accettazione o come tentare di dare risposta?

No, se alle richieste di aiuto tipo questa inseriamo _regione_manuale: tutte_ cosi come aveva fatto @luciaroma... Io direi di no, però parliamone...

**Comment by luciaroma** [comment]
io ho pensato di inserirla perché ho letto di diversi medici che hanno risposto ad appelli spostandosi da una regione all'altra, dunque ho pensato fosse utile si vedesse anche in altre regioni.

**Comment by chiccap** [comment]
Io lascerei geoloc su Alessandria come per le segnalazioni di questo tipo accettate in precedenza! In generale questo tipo di richieste di aiuto le manderei in bollettino senza investire del budget perchè sono un tipo di notizie che viaggia molto su canali specializzati, è giusto rilanciarle ovviamente! Decidiamolo però insieme!

**Comment by luciaroma** [comment]
ok va bene, mi hai convinta :-)

**Comment by luciaroma** [comment]
@cristigalas @favoeva @chiccap possiamo segnalare alla piattaforma di Alessandria? Così spingiamo un pò questa richiesta?

**Comment by cristigalas** [comment]
> @cristigalas @favoeva @chiccap possiamo segnalare alla piattaforma di Alessandria? Così spingiamo un pò questa richiesta?

Ho controllato e l’hanno pubblicata anche loro... forse potremmo chiedere a loro se hanno altre richieste di aiuto, ma dal sito non mi pare...

**Comment by claudiamazzantiact** [comment]
L'aggiornamento della segnalazione sul sito dell'ASL è del 31/03. Sono passati 9 giorni. Credo che se non ci siano stati nuovi appelli, questa segnalazione è un po' datata. Che ne pensate se li contattassimo per capire se è stato risolto questo bisogno e come diceva @cristigalas magari capiamo se ne hanno altri?

**Comment by cristigalas** [comment]
@claudiamazzantiact ok, provo a contattarli io su Slack...
